src/main/java/org/entando/kubernetes/model/bundle/processor/DirectoryProcessor.java

```diff
@@ -79,7 +79,7 @@ public ComponentType getSupportedComponentType() {
                         rootDirectoryAction));
             }
         } catch (IOException e) {
-            throw new EntandoComponentManagerException("Error reading bundle", e);
+            throwMeaningfulException(e);
```

Comment:
@firegloves hiding the throw may hide it from analyzers as java has no semantic for the case.

Consider this alternative:
`throw buildComponentManagerException(..)`